app/views/patients/index.html.erb

```diff
@@ -1,24 +1,29 @@
-<p style="color: green"><%= notice %></p>
 <div class= "container ">
-<h1>Pacientes</h1>
-<%= link_to "Nuevo paciente", new_patient_path,class:"btn btn-outline-secondary btn-sm"%>
-
-<table class="table table-striped"> 
-  <thead> 
-    <tr> 
-      <th>Nombre</th>
-      <th>Apellido</th>
-    </tr>
-  </thead>
-  <tbody>
-    <% @patients.each do |patient| %>
+  <p style="color: green"><%= notice %></p>
+  <blockquote class="blockquote text-center">
+    <h1>PACIENTES</h1>
+    <%= link_to "Nuevo paciente", new_patient_path,class:"btn btn-outline-secondary btn-sm"%>
+  </blockquote>
+  <table class="table table-striped">
+    <thead>
+      <tr>
+        <th></th>
+        <th>Nombre</th>
+        <th>Apellido</th>
+        <th>Fecha</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @patients.each do |patient| %>
       <tr>
```

Comment:
<tr> tag and its content should be indented, because is into "patients.each" block
```suggestion
        <tr>
```